heartbeat/simple_heart.go

```diff
@@ -45,11 +46,13 @@ func NewSimpleHeart(id, location string, interval time.Duration, pool *redis.Poo
 
 		pool: pool,
 
-		closer: make(chan struct{}),
-		errs:   make(chan error),
+		closer:   make(chan struct{}),
+		errs:     make(chan error, 1),
```

Comment:
If it's not buffered and an error occurs in the first touch, `NewSimpleHeart` will block indefinitely.